### DISCUSSION TITLE
How to re-enable windows 11 update service in winSxS?

### DISCUSSION THREAD

**Discussion by Andru115** [ntdevlabs/tiny11builder#218] [Q&A]
I installed tiny11coremaker and i just found out it removes winSxS so microsoft update is not included, any help will be appreciated!

**Comment by Miiraak** [comment]
Hi mate, it's mabye, a little late but...

Accordingly to the docs, you can not use coreMaker if you want to upgrade your system, as you said, it remove WinSxS, and it's not possible to re-enable removed features.

Use tiny11maker.ps1 instead 🐍

**Reply by Andru115** [reply]
No worries lad, i appreciate the help anyway, i used the ntdev tiny11 and it works flawlessly, thanks for the help lad. I appreciate it
